style.css

```diff
@@ -0,0 +1,635 @@
+
+hr {
+  color: rgb(206, 204, 204);
+}
+
+a {
+  text-decoration: none;
+  cursor: pointer;
+  color: #272a31;
+}
+
+a:active {
+  color: #ec5242;
+}
+
+a:hover {
+  color: #ec5242;
+}
+
+ul {
+  padding: 0;
+  margin: 0;
+}
+
+.d-flex {
+  display: flex;
+  flex-direction: row;
+}
+
+.space-between {
+  justify-content: space-between;
+}
+
+.align-center {
+  align-items: center;
+  text-align: center;
+}
+
+.menu {
+  background-color: #272a31;
+  color: #d3d3d3;
+  font-size: 25px;
+  height: 100vh;
+  z-index: 999;
+  align-items: center;
+  justify-content: center;
+  display: none;
+}
+
+.menu-close {
+  padding-top: 30px;
+  font-size: 35px;
+  cursor: pointer;
+}
+
+.menu-links {
+  width: 100%;
+  height: 80%;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.menu-links li {
+  list-style-type: none;
+}
+
+.nav-bar li {
+  list-style-type: none;
+  padding: 0 16px;
+}
+
+.partners-link li {
+  list-style-type: none;
+  margin: 10px 20px;
+}
+
+.menu-links li:last-child {
+  border: 1px solid;
+  padding: 5px 10px;
+}
+
+.program-footer a {
+  color: #d3d3d3;
+}
+
+.menu-links a {
+  color: #d3d3d3;
+}
+
+.menu-links a:hover {
+  color: #ec5242;
+}
+
+.nav-bar {
+  position: fixed;
+  padding: 20px 80px;
+  box-shadow: 0 0 10px #d3d3d3;
+  width: 100%;
+  top: 0;
+  z-index: 99;
+  background-color: #fff;
+}
+
+.nav-bar li:last-child {
+  border: 2px solid #ec5242;
+  width: 150px;
+  max-width: 150px;
+  padding: 0;
+}
+
+.intro {
+  position: relative;
+  background-color: rgba(76, 175, 80, 0.1);
+  width: 100%;
+  padding-top: 60px;
+}
+
+.intro::before {
+  content: '';
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  background-image: url('./images/coding-world.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  opacity: 0.1;
+}
+
+.p-heading {
+  font-size: 35px;
+  font-weight: 400;
+  color: #ec5242;
+}
+
+.color-gradient {
+  background: -webkit-linear-gradient(#ec5242, rgb(146, 40, 40), #ec5242);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+.intro-text {
+  position: relative;
+  padding-top: 80px;
+  padding-bottom: 80px;
+}
+
+.intro-text h1 {
+  font-size: 50px;
+  color: #ec5242;
+  font-weight: 900;
+  word-spacing: 10px;
+}
+
+.p-text {
+  width: 100%;
+  max-width: 600px;
+  padding-left: 20px;
+  flex-wrap: wrap;
+  margin: 40px 0;
+  border: 2px solid #fff;
+  background-color: #d3d3d334;
+}
+
+.intro-date {
+  font-weight: 900;
+  color: #272a31;
+}
+
+.main-program {
+  background-color: #272a31;
+  position: relative;
+  top: 0;
+}
+
+.section-heading {
+  color: #d3d3d3;
+  padding-top: 25px;
+  font-size: 30px;
+  width: 300px;
+  margin: 0 auto;
+}
+
+.section-hr {
+  width: 50px;
+  border: 1px solid;
+  margin: 10px auto;
+  color: #ec5242;
+  margin-bottom: 40px;
+}
+
+.main-program h4 {
+  color: #ec5242;
+}
+
+.program-card-container {
+  justify-content: center;
+  flex-wrap: wrap;
+}
```

Comment:
- [ ] The middle section article list in desktop view should be organized in a horizontal row as per the horizontal design. I don't think `flex-wrap: wrap` is a good property for this element. Kindly fix this. :+1: 

![image](https://user-images.githubusercontent.com/64894927/132693116-59dcef52-d80a-4ae0-b92b-91f0c29d2882.png)

![image](https://user-images.githubusercontent.com/64894927/132693168-b9ccf698-4646-4334-b3eb-61a9d5ad2d28.png)